src/Pages/Login/Login.js

```diff
@@ -0,0 +1,158 @@
+import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+import Header from '../../Component/HeaderComponent/Header';
+import Footer from '../../Component/FooterComponent/Footer';
+
+import './Login.scss';
```

Comment:
- scss 를 구분해주기 위해서 한 칸 띄우신 것 같은데, 위코드에서는 빈칸 대신 import 순서로 구분하는 컨벤션을 따릅니다!
```suggestion
import Footer from '../../Component/FooterComponent/Footer';
import './Login.scss';
```